lib/mayaUsd/fileio/shading/shadingModeRegistry.cpp

```diff
@@ -63,7 +80,23 @@ UsdMayaShadingModeRegistry::_GetExporter(const TfToken& name)
     UsdMaya_RegistryHelper::LoadShadingModePlugins();
     TfRegistryManager::GetInstance().SubscribeTo<UsdMayaShadingModeExportContext>();
     const auto it = _exportReg.find(name);
-    return it == _exportReg.end() ? nullptr : it->second;
+    return it == _exportReg.end() ? nullptr : it->second._fn;
+}
+
+const std::string&
+UsdMayaShadingModeRegistry::_GetExporterNiceName(const TfToken& name) {
+    UsdMaya_RegistryHelper::LoadShadingModePlugins();
+    TfRegistryManager::GetInstance().SubscribeTo<UsdMayaShadingModeExportContext>();
+    const auto it = _exportReg.find(name);
+    return it == _exportReg.end() ? _kEmptyString : it->second._niceName;
+}
+
+const std::string&
+UsdMayaShadingModeRegistry::_GetExporterDescription(const TfToken& name) {
+    UsdMaya_RegistryHelper::LoadShadingModePlugins();
+    TfRegistryManager::GetInstance().SubscribeTo<UsdMayaShadingModeExportContext>();
+    const auto it = _exportReg.find(name);
+    return it == _exportReg.end() ? _kEmptyString : it->second._description;
```

Comment:
Maybe refactor these three with a `_GetExporter()` that returns the new struct so we don't have almost identical code repeated three times?